build/rollup.js

```diff
@@ -76,6 +78,36 @@ const es = {
   banner: compiledLicense(Object.assign({includesVtt: true}, bannerData)),
   useStrict: false,
   format: 'es',
```

Comment:
Are we sure that we want to use the `es` format and not `cjs` for better compatibility?